games/Powermonger/powermonger.asm

```diff
@@ -70,6 +84,35 @@ _Start	;	A0 = resident loader
 		lea	(_resload,pc),a1
 		move.l	a0,(a1)
 
+	;get custom settings
+		move.l	a0,a1
+		lea	(_tags,pc),a0
+		jsr	(resload_Control,a1)
+
+	;decide whether to use fastmem
+		move.l	(_custom1,pc),d0
+		beq	.autofast
+		cmp.b	#1,d0
+		beq	.nofast
+		bra	.usefast
+.autofast
+	;auto -> use if 030+
```

Comment:
why starting at 30+? 
would it no be faster on 68000/20 too?

Reply:
I only have a 060 (and pistorm) HW to check, so I don't know. Based on TFX feedback the speed up method is worth it there (but I have no means of checking myself). 000 systems with true fast RAM are going to be very uncommon and I doubt it's worthwhile there (16-bit bus is really going to hurt), and it would for sure be a pessimisation if expmem is slow ram. Since 020 doesn't have a data cache I suspect it's also not going to be faster.

I tried to play it safe here, so 000/020 users don't get a slower game if they upgrade. Adventurous individuals can always opt in and try :)